src/style.css

```diff
@@ -2,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, sans-serif;
```

Comment:
The font-family 'Avenir' is used here. Ensure that this font is correctly loaded and available in the user's environment, or provide a fallback font that closely resembles 'Avenir'.